setup.py

```diff
@@ -13,6 +13,9 @@
 
 
 install_requires = [
+    'boto3>=1.15.0,<2',
+    'botocore>=1.18,<2',
+    'cloudpickle>=2.1.0,<3.0',
```

Comment:
I wanted to alphabetize these to make it easier to parse. I left our own libraries at the bottom but I can alphabetize them as well if we think that's better